manifest.php

```diff
@@ -38,12 +38,12 @@
     'label' => 'Item core extension',
     'description' => 'TAO Items extension',
     'license' => 'GPL-2.0',
-    'version' => '10.8.4',
+    'version' => '10.8.5',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => [
         'taoBackOffice' => '>=3.0.0',
         'generis' => '>=12.5.0',
-        'tao' => '>=38.5.0'
+        'tao' => '>=45.1.3'
```

Comment:
Just a friendly reminder, in case you decide to accept [that suggestion](https://github.com/oat-sa/tao-core/pull/2621#discussion_r462253756).

Don't forget to update the dependency version here as well.
```suggestion
        'tao' => '>=45.2.0'
```